src/test/java/com/epam/reportportal/junit/junit/SuiteProcessorTest.java

```diff
@@ -26,7 +26,9 @@
 import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.Description;
+import org.junit.runner.RunWith;
 
+@RunWith(UnitTestRunner.class)
```

Comment:
Without this, I couldn't figure out how to attach the ReportPortalListener

Reply:
@sbabcoc just put this class right into your test framework/test project.

Reply:
Without something like this, your unit tests don't actual appear to do anything in terms of Report Portal logging. Am I missing something?